.github/ISSUE_TEMPLATE/Post_RC_Code_Change.md

```diff
@@ -3,7 +3,7 @@ name: Post RC Code Change
 about: Template for Adding Code After RC is Cut requiring a new RC build
 title: Post RC Code Change Template
 labels: release
-assignees: djultratom, bischoffa
+assignees: TKDickson, bischoffa
```

Comment:
@TKDickson adam should be removed

Reply:
@timwright12 I thought about it, but I'd rather wait a few days to get Sarah's github info, and then be able to find and replace all of the Adam references with Sarah's username

Reply:
Can you drop a ticket in here for that so we can close the loop?

Reply:
Sure, #9828

Reply:
looks good!